inc/class-statify.php

```diff
@@ -79,6 +79,9 @@ public static function init() {
 
 		if ( defined( 'DOING_AJAX' ) && DOING_AJAX ) {
 			add_action( 'wp_ajax_nopriv_statify_track', array( 'Statify_Frontend', 'track_visit_ajax' ) );
+			if ( 0 === self::$_options['skip']['logged_in'] ) {
+				add_action( 'wp_ajax_statify_track', array( 'Statify_Frontend', 'track_visit_ajax' ) );
+			}
```

Comment:
Adding line 83 is sufficient, isn't it? The condition is checked by the `track_visit_ajax` function.

To jump out as early as possible, one could move the skip tracking check to the very first line of track_visit, right?

Reply:
You are right.

`skip_tracking` however  external action hooks before evaluating our logic, even with moved the login check up. 

Moving the skip-call up the `track_visit` method breaks  early return for the non-AJAX call.